packages/shared-ux/page/analytics_no_data/impl/src/analytics_no_data_page.tsx

```diff
@@ -20,14 +20,17 @@ export const AnalyticsNoDataPage = ({
   allowAdHocDataView,
 }: AnalyticsNoDataPageProps) => {
   const services = useServices();
-  const { kibanaGuideDocLink } = services;
+  const { kibanaGuideDocLink, customBranding } = services;
+  const { hasCustomBranding$, showPlainSpinner } = customBranding;
```

Comment:
This component should subscribe to `hasCustomBranding$` observable and pass on the resolved value as `showPlain Spinner` to the inner component

Reply:
Ok I think I might have refactored appropriately in [b51f0cf](https://github.com/elastic/kibana/pull/149253/commits/b51f0cf93f45355077b28d8f15908078df2b2e6b)